tests/AzureBlobStorageAdapterTest.php

```diff
@@ -423,20 +418,22 @@ public function testCopy($exception)
     {
         $blobClient = $this->createMock(BlobRestProxy::class);
         if ($exception) {
+            $responseInterface = $this->createMock(ResponseInterface::class);
+            $exceptionClass = new ServiceException($responseInterface);
             $blobClient->expects(self::any())
                 ->method('copyBlob')
-                ->willThrowException(new UnableToCopyFile(""));
+                ->willThrowException($exceptionClass);
             $this->expectException(UnableToCopyFile::class);
         } else {
             $blobClient->expects(self::any())
                 ->method('copyBlob')
                 ->willReturn(null);
-            $this->expectNotToPerformAssertions();
         }
 
-        //        $blobClient = BlobRestProxy::createBlobService('AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;DefaultEndpointsProtocol=http;BlobEndpoint=http://azurite:10000/devstoreaccount1;QueueEndpoint=http://azurite:10001/devstoreaccount1;TableEndpoint=http://azurite:10002/devstoreaccount1;');
         $service = new AzureBlobStorageAdapter($blobClient, $this->logger, 'default');
+        $response = true;
```

Comment:
I dont think this is correct, is there any other way to assert this?

Reply:
is correct the way, we can't corroborate the modification of the file, because all the external service is mocked, and the interface doesn't return anything, so we can assert the true or mark the test as risky